### DISCUSSION TITLE
NEXO Support

### DISCUSSION THREAD

**Discussion by kristophergeo** [BG-Software-LLC/SuperiorSkyblock2#2298] [Pending Feature Requests]
Support for NEXO for the custom blocks in SS2. Similar to how itemsadder and oraxen have
https://wiki.bg-software.com/superiorskyblock/overview/custom-blocks

NEXO: https://docs.nexomc.com/community-guides/api

**Comment by Lemvs** [comment]
@kristophergeo Nexo has been supported for some time.
@OmerBenGera you can close it.
